setup.py

```diff
@@ -1,80 +1,39 @@
 #!/usr/bin/env python
 
-import os
-import sys
 import glob
+import os
+from setuptools import setup  # type: ignore[import]
+import subprocess
 
-if sys.version_info < (3, 6):
-    print('ERROR: LTA requires at least Python 3.6+ to run.')
-    sys.exit(1)
-
-try:
-    # Use setuptools if available, for install_requires (among other things).
-    import setuptools  # type: ignore
-    setup = setuptools.setup
-except ImportError:
-    setuptools = None
-    from distutils.core import setup
-
-kwargs = {}
-
-current_path = os.path.dirname(os.path.realpath(__file__))
-
-with open(os.path.join(current_path, 'lta', '__init__.py')) as f:
-    for line in f.readlines():
-        if '__version__' in line:
-            kwargs['version'] = line.replace('"', "'").split('=')[-1].split("'")[1]
-            break
-    else:
-        raise Exception('cannot find __version__')
-
-with open(os.path.join(current_path, 'README.md')) as f:
-    kwargs['long_description'] = f.read()
+subprocess.run(
+    "pip install --upgrade pip".split(),
+    check=True,
+)
+subprocess.run(
+    "pip install git+https://github.com/WIPACrepo/wipac-dev-tools.git".split(),
+    check=True,
+)
+from wipac_dev_tools import SetupShop  # noqa: E402  # pylint: disable=C0413
 
-if setuptools is not None:
-    # If setuptools is not available, you're on your own for dependencies.
-    install_requires = [
-        'coverage>=4.4.2',
-        'PyJWT',
-        'pymongo',
-        'requests',
-        'requests_toolbelt',
-        'requests-futures',
-        'sphinx>=1.4',
-        'tornado>=5.1'
-    ]
-    kwargs['install_requires'] = install_requires  # type: ignore
-    kwargs['zip_safe'] = False  # type: ignore
+shop = SetupShop(
+    "lta",
+    os.path.abspath(os.path.dirname(__file__)),
+    ((3, 6), (3, 10)),
```

Comment:
I'm not sure how the GH Action will act with grabbing 3.10. But I see there's an issue

Reply:
running the bash lines locally, I get `3.6,3.7,3.8,3.9,3.10` so I'm inclined to think that this is because 3.10 has not been released. Or there's a bug in the tool I use (https://github.com/actions/setup-python).

Reply:
relevant: https://github.com/actions/setup-python/issues/177

Reply:
https://github.com/WIPACrepo/lta/pull/200#discussion_r640749447